rlbot_flat/src/lib.rs

```diff
@@ -50,8 +50,55 @@ impl From<flat::InterfaceMessage> for flat::InterfacePacket {
         flat::InterfacePacket { message }
     }
 }
+
 impl From<flat::CoreMessage> for flat::CorePacket {
     fn from(message: flat::CoreMessage) -> Self {
         flat::CorePacket { message }
     }
 }
+
+macro_rules! from_render_message {
+    ( $( $t:ident ),* ) => {
+        $(
+        impl From<flat::$t> for flat::RenderMessage {
+            fn from(value: flat::$t) -> Self {
+                Self {
+                    variety: flat::RenderType::$t(Box::new(value)),
+                }
+            }
+        }
+        )*
+    };
+}
+
+from_render_message!(Line3D, PolyLine3D, String2D, String3D, Rect2D, Rect3D);
+
+impl From<flat::Vector3> for flat::RenderAnchor {
+    fn from(value: flat::Vector3) -> Self {
+        Self {
+            world: value,
+            relative: None,
+        }
+    }
+}
```

Comment:
Maybe we should consider something like this instead? It would also remove the need for [this](https://github.com/RLBot/rust-interface/pull/7/files#diff-990230c90d31afe8d4f3a313b01e3f63e2f78c1c1b368de5136b1b160cae17a1R35-R52)

```suggestion
impl flat::RenderAnchor {
    pub fn from_vec(value: impl Into<flat::Vector3>) -> Self {
        Self {
            world: value.into(),
            relative: None,
        }
    }
}
```

Reply:
I don't like it because `RenderAnchor::from_vec(v)` is so much longer than `v.into()` 😅 And it would be inconsistent with `From<CarAnchor>` and `From<BallAnchor>` for relative.